src/main/java/uk/gov/hmcts/divorce/solicitor/event/SolicitorStatementOfTruthPaySubmit.java

```diff
@@ -80,6 +87,23 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             details.getId()
         );
 
+        log.info("Setting dummy payment to mock payment process");
+        if (caseData.getPayments() == null || caseData.getPayments().isEmpty()) {
+            List<ListValue<Payment>> payments = new ArrayList<>();
+            payments.add(
+                ListValue
+                    .<Payment>builder()
+                    .value(solicitorSubmitPetitionService.getDummyPayment(orderSummary))
+                    .build());
+            caseData.setPayments(payments);
+        } else {
+            caseData.getPayments().add(
+                ListValue
```

Comment:
isn't there a `new ListValue(item)`? We might want a static helper method in future: `listValue(item)`